components/ide-metrics/BUILD.yaml

```diff
@@ -0,0 +1,28 @@
+packages:
+  - name: app
+    type: go
+    deps:
+      - components/common-go:lib
+      - components/ide-metrics-api/go:lib
+    srcs:
+      - "**/*.go"
+      - "go.mod"
+      - "go.sum"
+    env:
+      - CGO_ENABLED=0
+      - GOOS=linux
+    config:
+      packaging: app
```

Comment:
We need change Version here, see also https://github.com/gitpod-io/gitpod/blob/8b9fd62f7bbdabb06b45874bc23a291fdf8e97a8/components/ws-daemon/BUILD.yaml#L20